hexrdgui/main_window.py

```diff
@@ -1562,7 +1560,12 @@ def on_action_hedm_import_tool_triggered(self):
         self.simple_image_series_dialog.show()
 
     def on_action_llnl_import_tool_triggered(self):
-        self.llnl_import_tool_dialog.show()
+        dialog = self.llnl_import_tool_dialog
+        dialog.show()
+        # Always assume Physics Package is needed for LLNL import
+        dialog.ui.complete.clicked.connect(
+            lambda: self.on_action_include_physics_package_toggled(True)
```

Comment:
This would create a new connection every time the LLNL import tool is shown. I'll move this to `setup_connections()` instead. Otherwise, this could get triggered multiple times.